src/interface/mesh_data.hpp

```diff
@@ -406,92 +354,49 @@ class MeshData {
     return PackVariablesAndFluxesImpl(nullptr, nullptr, sparse_ids);
   }
   // No nothing
-  const auto &PackVariablesAndFluxes(PackIndexMap &map, vpack_types::StringPair &key) {
-    return PackVariablesAndFluxesImpl(&map, &key);
-  }
-  const auto &PackVariablesAndFluxes(vpack_types::StringPair &key) {
-    return PackVariablesAndFluxesImpl(nullptr, &key);
-  }
   const auto &PackVariablesAndFluxes(PackIndexMap &map) {
     return PackVariablesAndFluxesImpl(&map, nullptr);
   }
   const auto &PackVariablesAndFluxes() {
     return PackVariablesAndFluxesImpl(nullptr, nullptr);
   }
 
-  // As above, no variatics because they shadow each other.
+  // As above, NO TEMPLATE ARGUMENTS, as they shadow each other
```

Comment:
I'm a bit confused by this comment, we still have the `Elem` template arguments...

Reply:
Oops. This is from my time when I was trying to debug. This comment is wrong and I'll change it back.